katello-attach-subscription

```diff
@@ -1455,7 +1479,8 @@ end
 # count the total number of subs available, consumed and free
 def getsubdetailfor(sub_name)
   detail_hash = {"consumed" => 0, "free" => 0}
-  search_options = KatelloAttachSubscription::Utils.search_args(name: sub_name)
+  search_params = 'name="'+sub_name+'"'
+  search_options = KatelloAttachSubscription::Utils.search_args(search_params)
```

Comment:
this will make `search_options` `"name="the_subname""` which sounds wrong to me?

Reply:
doesn't become 'name="the_subname"'?
From my test it always works using that piece of code

Reply:
So this is what I get with current master:

```
irb(main):002:0> sub_name = 'something special'
=> "something special"
irb(main):003:0> puts KatelloAttachSubscription::Utils.search_args(name: sub_name)
name="something special"
=> nil
irb(main):004:0> search_params = 'name="'+sub_name+'"'
=> "name=\"something special\""
irb(main):005:0> puts KatelloAttachSubscription::Utils.search_args(search_params)
"name="something special""
=> nil
```

Note the doubled quoting in the last line.

Reply:
but maybe I know what problem you were seeing, let me test something

Reply:
nope, not what I thought. please show me how it failed for you :)